types/key/nl.go

```diff
@@ -126,6 +137,12 @@ func (k NLPublic) MarshalText() ([]byte, error) {
 	return toHex(k.k[:], nlPublicHexPrefix), nil
 }
 
+// CLIString returns a marshalled representation suitable for use
```

Comment:
also spell out what this means concretely (which prefix it uses instead of which other prefix)

and update docs on the other string method

Reply:
Done.